Makefile

```
@@ -9,19 +9,16 @@ RELEASE = dev-t4c-manager
 NAMESPACE = t4c-manager
 SESSION_NAMESPACE = t4c-sessions
 PORT ?= 8080
-
+CAPELLA_VERSIONS = 5.0.0 5.2.0 6.0.0
+T4C_CLIENT_VERSIONS = 5.2.0 6.0.0
 CAPELLA_DOCKERIMAGES = $(MAKE) -C capella-dockerimages PUSH_IMAGES=1 DOCKER_REGISTRY=$(LOCAL_REGISTRY_NAME):$(REGISTRY_PORT)
 
 # Adds support for msys
 export MSYS_NO_PATHCONV := 1
 
-build: backend frontend docs
-
-build-capella:
-	$(CAPELLA_DOCKERIMAGES) capella/remote capella/readonly
+build: backend frontend docs capella
```

```suggestion
build: backend frontend docs
```

I'd not include it here. At least, we should have a separate target to build images except the Capella Docker images.